src/services/Testing/DummyData.js

```diff
@@ -285,6 +285,25 @@ export const dummyItems = {
   ],
 };
 
+export const imageData = {
+  dataUrl:
```

Comment:
did you compute this value for an image and then place it here?

Reply:
Yep